assets/prototype/domain/eventEditor/containers/mutations/useDatetimeMutator.js

```diff
@@ -0,0 +1,166 @@
+import useRelations from '../../../../infrastructure/services/relations/useRelations';
+import { queries } from '../queries';
+import useEventId from '../queries/useEventId';
+
+const { GET_DATETIMES, GET_TICKETS } = queries;
+/**
+ */
+const useDatetimeMutator = () => {
+	const { updateRelations, addRelation, removeRelation, dropRelations } = useRelations();
+	const eventId = useEventId();
+
+	const options = {
+		query: GET_DATETIMES,
+		variables: {
+			where: {
+				eventId,
+			},
+		},
+	};
+
+	const mutator = (mutationType, input) => {
+		/**
+		 * @todo update optimisticResponse
+		 */
+		let variables, optimisticResponse, update;
+
+		variables = {
+			input: {
+				clientMutationId: `${mutationType}_DATETIME`,
+				...input,
+			},
+		};
+
+		update = update = (
+			proxy,
+			{
+				data: {
+					[`${mutationType.toLowerCase()}Datetime`]: { datetime },
```

Comment:
Maybe we can extract this and give it a name, to have a better understanding/ readability.

Reply:
Good idea

Reply:
Done